src/pages/iou/request/step/getAddressFormHeight/index.native.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * get the height of Form view in GooglePlacesAutocomplete.
+ * @param {Number} windowHeight the height of windows
+ * @param {Boolean} isOffline Whether network is offline
+ * @returns {Object} styles for the Form height.
+ */
+function getAddressFormHeight(windowHeight, isOffline) {
+    return {
+        maxHeight: windowHeight - 175 - (isOffline ? 26 : 0),
```

Comment:
1. I don't think using hardcoded values is an acceptable approach. What are 175 and 26 in this case?
2. Why does address form height depends on whether we're offline or not?